app/components/file_attachment_list/file_attachment_list.js

```diff
@@ -100,7 +99,8 @@ export default class FileAttachmentList extends Component {
                 }
 
                 if (cache) {
-                    uri = cache.path;
+                    const prefix = Platform.OS === 'android' ? 'file://' : '';
```

Comment:
Is this something that needs to be fixed on the release branch? I forget where those changes were done?

Reply:
I think it already was, wasn’t it?

Reply:
Yup, it's there on the release branch.